src/parsers/open-api2.js

```diff
@@ -45,12 +81,15 @@ function buildAjvValidator(ajvConfigBody, formats, keywords){
     return ajv;
 }
 
-function buildResponseBodyValidation(schema, swaggerDefinitions, originalSwagger, currentPath, currentMethod, options, statusCode) {
+function buildResponseBodyValidation(responses, swaggerDefinitions, originalSwagger, currentPath, currentMethod, options, statusCode) {
```

Comment:
use the same function for buildResponseBodyValidation and buildRequestBodyValidation